### DISCUSSION TITLE
I have a lot of outliers in my project, I want to get the probability of each sentence belonging to each topic, If I use topics,probs= topic_model.fit_transform(sentences,embeddings=embeddings) the probs obtained by this method will contain every sentence (including outliers). Is there any way to extract these outliers, and do these outliers also have topic probability proportions

### DISCUSSION THREAD

**Discussion by superseanyoung** [MaartenGr/BERTopic#2220] [General]
topic_model = BERTopic(embedding_model=transformer_model, 
                           min_topic_size=3, 
                           verbose=True,
                           umap_model=umap_model,
                           hdbscan_model=hdbscan_model,
                           ctfidf_model=ctfidf_model,
                           representation_model=representation_model,
                           #top_n_words=10,
                           #min_topic_size=10,
                           #nr_topics=None,
                           #low_memory=False,
                           calculate_probabilities=True
                          )
    topics,probs= topic_model.fit_transform(sentences,embeddings=embeddings)

**Comment by superseanyoung** [comment]
In the obtained probs, I do not want to get sentences belonging to outliers, how should I post them

**Reply by MaartenGr** [reply]
Have you checked out the [documentation of BERTopic](https://maartengr.github.io/BERTopic/)? You should be able to find the solution there. 

To help you out, please refer to the [FAQ](https://maartengr.github.io/BERTopic/faq.html#how-do-i-reduce-topic-outliers) about this specific question.

**Reply by superseanyoung** [reply]
I update the topic after reducing the outliers, whether this will change the previous probs

**Reply by MaartenGr** [reply]
The probabilities should stay the same after reducing outliers if you do not merge topics together.
